interoperability_report.py

```diff
@@ -0,0 +1,878 @@
+#!/usr/bin/python
+
+import time
+import re
+import pexpect
+import argparse
+from junitparser import TestCase, TestSuite, JUnitXml, Attr, Failure
+from multiprocessing import Process, Queue, Manager, Event
+from datetime import datetime
+import tempfile
+from os.path import exists
+
+from utilities import ReturnCode
+from testSuite import rtps_test_suite_1
+
+def log_message(message, verbosity):
+    if verbosity:
+        print(message)
+
+def subscriber(
+        name_executable,
+        parameters,
+        testCase,
+        time_out,
+        producedCode,
+        samplesSent,
+        subscriber_finished, publisher_finished,
+        file,
+        verbosity):
+
+    """ Run the executable with the parameters and save
+        the error code obtained
+
+        name_executable     : name of the ShapeApplication to run
+                              as a Subscriber
+        parameters          : QOS to use with the Shape Application
+        testCase            : testCase is being tested
```

Comment:
```suggestion
        test_name            : name of the test that is being tested
```

is this the test name?